scripts/deterministic.js

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
As this is taken from https://github.com/catapult-project/catapult/blob/183d99e38bfe9d296913b357c80599577056700f/web_page_replay_go/deterministic.js we should include the copyright statement and license. I would suggest creating a subdirectory named 'catapult' and putting the LICENSE file from https://github.com/catapult-project/catapult/blob/master/LICENSE in there.

Reply:
Done